src/motioncore/base/backend.h

```diff
@@ -142,6 +142,26 @@ class Backend : public std::enable_shared_from_this<Backend> {
 
   SharePointer BooleanGmwInput(std::size_t party_id, std::vector<BitVector<>>&& input);
 
+  /// \brief share constant values (publicly known before circuit evaluation) as Boolean gmw share
```

Comment:
The description of `ConstantBooleanGmwInput` suggests that a constant value is known to only one party. The definitions I know of use the word "constant" for global constants, so all parties need to know them. In this case I would rather use something like "plaintext". BTW, where do you want to use such values?

Reply:
`ConstantAsBooleanGmwInput` is a upper class for `ConstantAsBooleanGmwInputGate`.
Please see [Discussion about ConstantAsBooleanGmwInputGate](https://github.com/encryptogroup/MOTION/pull/29#discussion_r951141103)

Reply:
answered there